controllers/experiencias/conseguirExperienciaID.js

```diff
@@ -1,31 +1,47 @@
 const conexionMysql = require("../../DB/conexionMysql");
 
 /**
- * Selecciona de la tabla de experiencias un único registro especificando su ID. 👍 
- * @param {} req 
- * @param {*} res 
- * @param {*} next 
+ * Selecciona de la tabla de experiencias un único registro especificando su ID. 👍
+ *
+ * @param req
+ * @param {any} res
+ * @param {any} next
  */
 async function conseguirExperienciaID(req, res, next) {
-    let conexion;
-    try {
-        conexion = await conexionMysql();
-        const { id } = req.params;
-        const [result] = await conexion.query(
-            `
+  let conexion;
+  try {
+    conexion = await conexionMysql();
+    const { id } = req.params;
+    const [result] = await conexion.query(
+      `
             SELECT * FROM experiencias WHERE id=?
             `,
-            [id]);
+      [id]
+    );
 
-        res.send({
-            status: 'Ok',
-            data: result
-        });
-    } catch (err) {
-        next(err)
-    } finally {
-        if (conexion) conexion.release();
-    }
+    let [single] = result;
```

Comment:
Esta variable es innecesaria y en todo caso debería ser una constante

Reply:
Si quito single me da error
{
    "status": "error",
    "message": "single is not defined"
}

Reply:
Te da error porque tienes que introducir result en el objeto con el que responde la petición. Esto debería funcionar:
```javascript
const [result] = conexion.query(
        `
        SELECT * FROM experiencias where id=?
        `, [id]
    )
res.send({
      status: "Ok",
      data: {
        result,
        fotos,
        } 
    });
```